.travis.yml

```diff
@@ -3,13 +3,21 @@ sudo: required
 language: bash
 services: docker
 
-env:
-  - DOCKER_IMAGE=nginx-rtmp
+jobs:
+  include:
+    - env: 
+      - DOCKER_FILE=Dockerfile
+      - DOCKER_TAG=latest
+      - DOCKER_IMAGE=nginx-rtmp 
+    - env:
+      - DOCKER_FILE=DockerfileCudaSupport
```

Comment:
Can this be renamed to `Dockerfile.cuda`? Just a convention I use.

Reply:
Yes. sure! :)